crates/nu-command/src/filters/get.rs

```diff
@@ -152,6 +158,13 @@ If multiple cell paths are given, this will produce a list of values."#
                 since: Some("0.105.0".into()),
                 expected_removal: None,
                 help: Some("Cell-paths are now case-sensitive by default.\nTo access fields case-insensitively, add `!` after the relevant path member.".into())
+            },
+            DeprecationEntry {
+                ty: DeprecationType::Flag("ignore-errors".into()),
+                report_mode: ReportMode::FirstUse,
+                since: Some("0.106.0".into()),
+                expected_removal: None,
+                help: Some("This flag has been renamed to `--optional (-o)` to better reflect its behavior.".into())
```

Comment:
I can try to extract out the "rename" parts of #15869 into a PR that we can use for this